flake.nix

```diff
@@ -171,7 +171,19 @@
               source ${writeScript ./nix/shell.sh}
             '';
           });
-
+        devShells.ktls = pkgs.mkShell {
+          # Intended for kTLS.
+          # This is not an over-ride like the other shells.
+          buildInputs = [ pkgs.cmake openssl_1_1_1 ];
+          S2N_LIBCRYPTO = "openssl-1.1.1";
+          OPENSSL_1_1_1_INSTALL_DIR = "${openssl_1_1_1}";
```

Comment:
Should this be awslc instead? If we're only going to use one libcrypto, it should probably be awslc.

Reply:
updated; I think this was done before the awslc flake was done.